tools/autograph-monitor/constants.go

```diff
@@ -1,18 +1,92 @@
 package main
 
-// autographDevRootHash is the SHA2 hash of the cacert in the
+// autographDevRootHashes is the SHA2 hashes of the cacerts in the
 // autograph dev config:
 // https://github.com/mozilla-services/autograph/blob/b3081068f4a9c0c1de02150432f2d02887dd6722/autograph.yaml#L113-L126
 // used on the normandy and remote settings development servers
-const autographDevRootHash = `5E:36:F2:14:DE:82:3F:8B:29:96:89:23:5F:03:41:AC:AF:A0:75:AF:82:CB:4C:D4:30:7C:3D:B3:43:39:2A:FE`
+var autographDevRootHashes = []string{
```

Comment:
This can be dropped now that we're calculating these hashes at runtime, yeah?

Reply:
Ohhh, I see

Reply:
I think I've been looking at this code for way too long. :dizzy: